### ISSUE TITLE
Secondary structure minimap in puzzlemaker fails to indicate solved-ness.

### ISSUE THREAD

**Issue by everyday847** [eternagame/EternaJS#513]
Reported by jnicol.

![image](https://user-images.githubusercontent.com/5008491/97826378-4c7e5b00-1c76-11eb-82c8-c1b62bb1d99d.png)

"The shape should have 4 pairs. The mini map shows unsolved and no pair display at bottom GC pair count"

It looks like it's, in particular, failing to update the TARGET STRUCTURE -- i.e., the pairs are red in the minimap because they "shouldn't form" in the target (also explains the fact that the delta is negative).

**Comment by everyday847** [comment]
Might start in one of these commits but it also might start earlier -- there are a lot of commits where puzzlemaker SPECIFICALLY is broken:

```
01cd4850809bfa3cc512ac0c4dbd4fe4d3132e15
664377181bbd84cd827de5d383559ee0759c1779
988b51d68b1595d07f7bbc185a9f968acacbf707
799149576997b79565b2ddcfaed878b9cfe308eb
cda86635fe626e3b0f6b1391bd4059e7b9310a5f
4c6539cfbcc2e1961d7a0bf5688254a9f7ec0d9b
6a01a3eb528e4df36223a07cd13b5cc25c58d117 
24153174016ab816ae3dbb8664f8034dec4caa87
8982a892f9ce1939b483032c49f89b55139d6e31
19df3e23013fa9a3bad182d9c52f2a570bb885f2
ffccd400d7140072fa2016c3dc8af40da614490c
7a4a1e51c31d1fcd5f313ea81688802e4efb0d5f
aba0edc1d0837565852708bd6c45224efca954cd
dccd48afa91f04b84b0a9e382a7f57975311e57e
```

**Comment by everyday847** [comment]
Resolved in 6bb5b71dcc
